src/topology/algebra/module.lean

```diff
@@ -623,4 +713,78 @@ by { ext x, refl }
 @[simp] theorem symm_symm_apply (e : M ≃L[R] M₂) (x : M) : e.symm.symm x = e x :=
 rfl
 
+/-- Create a `continuous_linear_equiv` from two `continuous_linear_map`s that are
+inverse of each other. -/
+def of_inverse (f₁ : M →L[R] M₂) (f₂ : M₂ →L[R] M) (h₁ : function.left_inverse f₂ f₁)
+  (h₂ : function.right_inverse f₂ f₁) :
+  M ≃L[R] M₂ :=
+{ to_fun := f₁,
+  continuous_to_fun := f₁.continuous,
+  inv_fun := f₂,
+  continuous_inv_fun := f₂.continuous,
+  left_inv := h₁,
+  right_inv := h₂,
+  .. f₁ }
+
+@[simp] lemma of_inverse_apply (f₁ : M →L[R] M₂) (f₂ h₁ h₂ x) :
+  of_inverse f₁ f₂ h₁ h₂ x = f₁ x :=
+rfl
+
+@[simp] lemma symm_of_inverse (f₁ : M →L[R] M₂) (f₂ h₁ h₂) :
+  (of_inverse f₁ f₂ h₁ h₂).symm = of_inverse f₂ f₁ h₂ h₁ :=
+rfl
+
+section
+variables (R) [topological_space R] [topological_module R R]
+
+/-- Continuous linear equivalences `R ≃L[R] R` are enumerated by `units R`. -/
+def equiv_units : units R ≃ (R ≃L[R] R) :=
```

Comment:
As below, I'm not certain about this name. Perhaps `units_equiv_aut`, `units_equiv_linear_equiv_self`, ...?